.babelrc

```diff
@@ -1,20 +1,33 @@
 {
   "presets": [
-    "latest",
     "react",
     "stage-1"
   ],
   "env": {
     "development": {
       "presets": [
+        "latest",
         "react-hmre"
       ]
     },
     "production": {
+      "presets": [
```

Comment:
Why is this change now necessary? Is this change related to the move to Webpack 2?